src/features/move-columns/js/column-movable.js

```diff
@@ -306,6 +306,7 @@
             post: function ($scope, $elm, $attrs, uiGridCtrl) {
               function enableColumnMove(){
                   if ($scope.grid.options.enableColumnMoving && $scope.col.colDef.enableColumnMoving) {
+                    offAllEvents();
```

Comment:
So now the if and the else do this?  we can move that to a common spot?

its also really odd that they wouldn't do it if its already in the else.  Have you checked their blame to see what happend there?

Reply:
I mean, I think there was just the assumption that you're coming from a state where it wasn't enabled, so they must not be registered.  I can move the call to a common spot though

Reply:
I would like to be sure, if you can at least look to see if there is anything in the history.

Reply:
FWIW, it looks like @JeffRiggle made the original change to register the event during the $watch: https://github.com/GenesysPureConnect/ng-grid/commit/3e53f156a26f1f9e1ffa132d1f86959afd8a4e0f

And we later added the second watch as a bug fix (I think if you had enableColumnMoving: true, but the colDef version false, and then later made that true as well, it wouldn't update).  But yeah, unless Jeff has something else to say about it, it looks like we couldn't get into this state when the change was originally made.

Reply:
And this introduced the current bug: https://github.com/GenesysPureConnect/ng-grid/commit/8db03b4cc6448d94c24674155069d174a2d21290#diff-b95f05f2ddaa90b37e551c52fd9a3f8a